go/test/endtoend/vtgate/reservedconn/reconnect1/main_test.go

```diff
@@ -133,7 +133,7 @@ func TestServingChange(t *testing.T) {
 
 	// changing rdonly tablet to spare (non serving).
 	rdonlyTablet := clusterInstance.Keyspaces[0].Shards[0].Rdonly()
-	err = clusterInstance.VtctlclientProcess.ExecuteCommand("ChangeTabletType", rdonlyTablet.Alias, "replica")
+	err = clusterInstance.VtctldClientProcess.ExecuteCommand("ChangeTabletType", rdonlyTablet.Alias, "replica")
```

Comment:
Unrelated to this PR, but each of these test packages should be renamed to be descriptive instead of being called 1,2,3,4.